.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
```

Comment:
This `.github/workflows/main.yml` file you added doesn't seem relevant. Please remove.